src/middleware/isValidInvite.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken')
+const config = require('../config')
+const { AppError } = require('../lib')
+
+
+const verify = (req, res, next) => {
+	const authHeader = req.headers.authorization
+
+	if (!authHeader) return next(new AppError('No token provided', 401))
+
+	const token = authHeader.split(' ')[1]
+
+	return jwt.verify(token, config('TOKEN_SECRET'), (err, user) => {
+
+		if (err) return next(new AppError('Invalid token', 403))
```

Comment:
Could you share why we are making the status code change here?

I was thinking that 403 means the credentials are valid but there are not enough permissions to perform the specific action

Reply:
@jocrah fixed.